fltio.c

```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
Few things!
1. You didn't include comments! Always include comments.
2. Your code was good, but it only worked for five numbers!